lib/fastlane/actions/get_info_plist_value.rb

```diff
@@ -0,0 +1,51 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      GET_INFO_PLIST_VALUE_CUSTOM_VALUE = :GET_INFO_PLIST_VALUE_CUSTOM_VALUE
+    end
+
+    class GetInfoPlistValueAction < Action
+      def self.run(params)
+        require "plist"
+        begin
+          path = File.expand_path(params[:plist_path])
+	      plist = Plist::parse_xml(path)
+ 	      Actions.lane_context[SharedValues::INFO_PLIST_VALUE] = plist[params[:param_name]]
+        rescue
+          Helper.log.info "Unable to find plist file at #{path}".red
+ 	    end
+      end
+
+      def self.description
+        "returns value from Info.plist of your project as native Ruby data structures"
+      end
+
+      def self.available_options
+        [
+		  FastlaneCore::ConfigItem.new(key: :param_name,
```

Comment:
It seems like the code formatting still has some issues. Make sure to switch to a 2 space intend :+1: